docs/features/blocks/breadcrumbs.md

```diff
@@ -0,0 +1,150 @@
+---
+id: breadcrumbs
+title: "Blocks - Breadcrumbs"
+sidebar_label: Breadcrumbs
+description: This documentation provides information about breadcrumbs block.
+---
+
+This documentation provides information about the breadcrumb block
+that [Yoast SEO](https://yoast.com/wordpress/plugins/seo/) provides.
+
+## Filters
+
+### Changing the output {#breadcrumb_output}
+
+The `wpseo_breadcrumb_output` filter takes the output of the breadcrumb. Make sure not to empty the output, just replace
+or add data. An example:
+
+```php
+add_filter( 'wpseo_breadcrumb_output', [ $this, 'override_home_icon' ] );
+
+/**
+ * Replace the home text with an icon.
+ *
+ * @param  string $original_breadcrumbs The whole breadcrumbs html.
+ * @return string
+ */
+public function override_home_icon( string $original_breadcrumbs ): string {
+    $new_home = \str_replace( '>Home<', ' aria-label="Home"><i class="fa fa-homeicon" aria-hidden="true"></i><', $original_breadcrumbs );
+    return $new_home;
+}
+```
+
+### Changing a single link {#breadcrumb_single_link}
+
+The `wpseo_breadcrumb_single_link` allows changing of each link being put out by the Yoast SEO breadcrumbs class. It
+takes in the generated link and the source breadcrumb information. An example:
+
+```php
+add_filter( 'wpseo_breadcrumb_single_link', [ $this, 'override_home_text' ] );
```

Comment:
Same comment as above, but also you left the name of the other function, please adapt 🙂 

```suggestion
add_filter( 'wpseo_breadcrumb_single_link', 'remove_current_page' );
```